src/modules/rewards/services/arb-rebate-service.ts

```diff
@@ -268,18 +267,25 @@ export class ArbRebateService {
       };
     }
 
+    const inputTokenPrice = deposit.price.usd;
     const historicRewardTokenPrice = await this.marketPriceService.getCachedHistoricMarketPrice(
       DateTime.fromJSDate(deposit.depositDate).minus({ days: 1 }).toJSDate(),
       rewardToken.symbol.toLowerCase(),
     );
+
+    const inputAmountUsd = new BigNumber(deposit.amount)
+      .multipliedBy(inputTokenPrice)
+      .dividedBy(new BigNumber(10).pow(deposit.token.decimals));
```

Comment:
Nit: Is this something we do a lot in the code? We may want to consider putting this into a function

Reply:
Yes, there are some places where we do this. I think I can work on this on a separate PR to change it in all the required places